lib/fxpay/pay.js

```diff
@@ -79,7 +84,8 @@
   };
 
 
-  function processWebPayment(paymentWindow, payJwt, callback) {
+  function processWebPayment(paymentWindow, managePaymentWindow, payJwt,
```

Comment:
I am beginning to dislike option objects because it's so hard to keep track of what the valid arguments are. Meh. But anyway this function is only called in one place. I hate javascript.